library/Icingadb/Widget/ItemList/BaseCommentListItem.php

```diff
@@ -101,7 +104,17 @@ protected function assembleVisual(BaseHtmlElement $visual)
 
     protected function createTimestamp()
     {
-        return new TimeAgo($this->item->entry_time);
+        if ($this->item->expire_time) {
+            return Html::tag(
+                'span',
+                FormattedString::create(t("expires %s "), new TimeUntil($this->item->expire_time))
```

Comment:
What is the trailing space in the string for?

Reply:
Introduced mistakenly. I have removed them now.